core/blockchain.go

```diff
@@ -633,12 +633,6 @@ func (bc *BlockChain) startAcceptor() {
 		start := time.Now()
 		acceptorQueueGauge.Dec(1)
 
-		if err := bc.flattenSnapshot(func() error {
-			return bc.stateManager.AcceptTrie(next)
-		}, next.Hash()); err != nil {
-			log.Crit("unable to flatten snapshot from acceptor", "blockHash", next.Hash(), "err", err)
-		}
-
 		// Update last processed and transaction lookup index
```

Comment:
Any good reason to move/not move this?

Reply:
just for a smaller diff for this release